web_src/less/_repository.less

```diff
@@ -4,6 +4,12 @@
     // otherwise some part of the popup will be hidden by viewport boundary
     max-height: 45vh;
     max-width: 60vw;
+
+    & .ui.right {
+      // Override the .ui.attached.header .right:not(.dropdown) height: 30px;
```

Comment:
Could we have more comments about why this should be overridden and what problem is resolves ..... Without reading the original issue, readers have no knowledge about this

Reply:
```suggestion
      // Override `.ui.attached.header .right:not(.dropdown) height: 30px;` which would otherwise lead to
      // the status popup box having its height fixed at 30px. See https://github.com/go-gitea/gitea/issues/18498
```